airbyte-protocol/protocol-models/src/main/resources/airbyte_protocol/airbyte_protocol.yaml

```diff
@@ -74,45 +74,60 @@ definitions:
     properties:
       state_type:
         "$ref": "#/definitions/AirbyteStateType"
+      stream:
+        "$ref": "#/definitions/AirbyteStreamState"
+      global:
+        "$ref": "#/definitions/AirbyteGlobalState"
       data:
         description: "(Deprecated) the state data"
         type: object
         existingJavaType: com.fasterxml.jackson.databind.JsonNode
-      global:
-        "$ref": "#/definitions/AirbyteStateBlob"
-      streams:
-        type: array
-        items:
-          "$ref": "#/definitions/AirbyteStreamState"
 
   AirbyteStateType:
     type: string
     description: >
       The type of state the other fields represent.
       If not set, the state data is interpreted as GLOBAL and should be read from the `data` field for backwards compatibility.
-      GLOBAL means that the state should be read from `global` and means that it represents the state for all the streams.
-      PER_STREAM means that the state should be read from `streams`. Each item in the list represents the state for the associated stream.
+      GLOBAL means that the state should be read from `global` and means that it represents the state for all the streams. It contains one shared
+      state and individual stream states.
+      PER_STREAM means that the state should be read from `stream`. The state present in this field correspond to the isolated state of the
+      associated stream description.
     enum:
       - GLOBAL
       - PER_STREAM
```

Comment:
```suggestion
      - STREAM
      - LEGACY
```

Reply:
per @jdpgrailsdev [comment](https://docs.google.com/document/d/10cpbytxVzuwSpLtAA4fQeDoXRE17sY81h0bTrUPSE7s/edit?disco=AAAAamH4DGQ) let's add legacy

Reply:
also i thought we were just calling it `STREAM`, not `PER_STREAM`

Reply:
also good to mention in the description that <NOT-SET> is the same as LEGACY.

Reply:
Done